### DISCUSSION TITLE
Please add taking notes for each repositories

### DISCUSSION THREAD

**Discussion by salihmarangoz** [community/community#36242] [Other Feature Feedback, Questions, & Ideas]
I would like to take notes for the repositories that I starred. Now I am doing it manually here: https://github.com/salihmarangoz/salihmarangoz/blob/main/STARRED_REPOSITORIES.md

I think this would be useful because I start forgetting what they were for or if there were any problems, etc.

Also, it would be good to switch to a compact view of the list of items.

Thanks.

**Comment by jfullstackdev** [comment]
maybe it can be done by Lists, a new feature of GitHub, go to Stars and make a List, that way you can group related repos 

![image](https://user-images.githubusercontent.com/47092464/195993815-22d95218-1c70-486b-99a9-844e097cb25b.png)

and you can make a description of that list

**Reply by salihmarangoz** [reply]
It is possible to add description/notes for each group but not per repository. I would like to do it for each repository separately. Thanks btw

**Comment by Octocat5B** [comment]
I have a list from my GitHub but now í don't seen anything from my GitHub accounts that I started before, can you please help?

**Comment by ossanna16** [comment]
Hey there @salihmarangoz and thank you for the suggestion! I'm glad you found our community 🙂

**Comment by appletv850** [comment]
> I would like to take notes for the repositories that I starred. Now I am doing it manually here: https://github.com/salihmarangoz/salihmarangoz/blob/main/STARRED_REPOSITORIES.md
> 
> I think this would be useful because I start forgetting what they were for or if there were any problems, etc.
> 
> Also, it would be good to switch to a compact view of the list of items.
> 
> Thanks.

**Comment by mlisowsk** [comment]
Seconding OPs proposal. Using lists does not solve the issue: we want to have a (private) note for each individual repo, not the entire list.
I use lists to evaluate repos for use in my projects, so I end up having a list of a dozen repos each solving the same problem.
Why did I star this repo? What were pros and cons of that repo?
Examples: "that repo is great, but heavyweight. Or it has a dependency I don't like or it maybe doesn't work in the environment or maybe it's abandoned."
Often repos have similar names, then it becomes even more difficult to remember - unless you write it down somewhere.
Ideally the note should be displayed when listing the starred repos (or at least the first line of note).

Again, the note should be a property of the starred repo, not a list. We should see the note for listed and unlisted starred repos.
